db-service/lib/cqn2sql.js

```diff
@@ -197,7 +197,7 @@ class CQN2SQLRenderer {
       sql = `SELECT`
     if (distinct) sql += ` DISTINCT`
     if (!_empty((x = columns))) sql += ` ${x}`
-    if (!_empty((x = from))) sql += ` FROM ${this.from(x)}`
+    if (!_empty((x = from))) sql += ` FROM ${this.from(x, q)}`
```

Comment:
btw: we don't always need to enhance args. this.cqn also gives access to top-level query

Reply:
As the view is replaced with a select the from is applied to nested select queries. So wile it is possible to find the correct parent query of a from in the `this.cqn` it will probably be more complicated.

Reply:
These are cases where using `this.cqn` is not appropriate